extensions/http-curl/tests/HTTPSiteToSiteTests.cpp

```diff
@@ -123,61 +123,66 @@ struct test_profile {
 void run_variance(std::string test_file_location, bool isSecure, std::string url, const struct test_profile &profile) {
   SiteToSiteTestHarness harness(isSecure);
 
-  SiteToSiteLocationResponder responder(isSecure);
+  SiteToSiteLocationResponder *responder = new SiteToSiteLocationResponder(isSecure);
```

Comment:
this is a short lived test, we don't care about memory leaks here. and we don't control stoppage of the web server, so we can avoid issues entirely by simply adding this to the heap and not concerning ourselves with scope.